3-ecommerce-db/projects/cyf-ecommerce-api/app.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const port = 3002;
+const bodyParser = require('body-parser');

```

Comment:
we didn't use `bodyParser` but you can try
you may need also `cors` package as well
please check our examples we have already worked earlier on

Reply:
i think 
you probably wanted to have here the whole content of server folder.
remove files:
- ./3-ecommerce-db/projects/cyf-ecommerce-api/app.js
- ./3-ecommerce-db/projects/cyf-ecommerce-api/package.json
- ./3-ecommerce-db/projects/cyf-ecommerce-api/package-lock.json
and move all file from ./server/ folder here.
then remove empty ./server/ folder.